src/main/java/am/ik/yavi/builder/ValidatorBuilder.java

```diff
@@ -118,6 +127,11 @@ public ValidatorBuilder(String messageKeySeparator) {
 		return (ValidatorBuilder<S>) this;
 	}
 
+	/**
+	 * @deprecated please use the copy constructor {@link #ValidatorBuilder(ValidatorBuilder)}
+	 * @return the cloned builder
+	 */
+	@Deprecated
 	public ValidatorBuilder<T> clone() {
 		final ValidatorBuilder<T> builder = new ValidatorBuilder<>(
 				this.messageKeySeparator);
```

Comment:
would be nice to use the copy constructer here?

Reply:
Yes this would make sense and remove duplicated code

Reply:
Fixed